addons/miscitems/CfgVehicles.hpp

```diff
@@ -0,0 +1,1042 @@
+class CfgVehicles {
+    class Item_Base_F;
+    class ACE_MiscItem_F: Item_Base_F {
+        vehicleClass = "ACE_Misc_Items";
+    };
+
+    class ACE_MiscItems_FMRadio_Item: ACE_MiscItem_F {
+        scope = 2;
+        scopeCurator = 2;
+        displayName = CSTRING(FMRadio_DisplayName);
+        author = ECSTRING(common,ACETeam);
```

Comment:
Shows how limited the author entry is. My policy is to add me as author if the config is made by me.